### ISSUE TITLE
Fix Card Size

### ISSUE THREAD

**Issue by ashutoshkrris** [ashutoshkrris/iRead#7]
### **Describe the bug**
In Homepage, the post cards are not of fixed size and look irregular.

### **To Reproduce**
1. Go to 'www.iread.ga'
2. Scroll down to 'Latest Posts'
3. See error

### **Expected behavior**
All the post card should be of same size so that they look nicer.

### **Screenshots**
![Screenshot (397)](https://user-images.githubusercontent.com/47353498/121852669-71a77180-cd0d-11eb-8272-81ff02bd68d3.png)

### **Additional context**

Note: If you want to work on an issue, you should check if it has already been assigned to anyone. If the issue is free you can comment `/assign` to get the issue assigned to you.
If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.

**Comment by Garima-7** [comment]
Assign it to me.

**Comment by ashutoshkrris** [comment]
Alright @Garima-7 
Assigning it to you!

**Comment by purohitmansi** [comment]
Please do assign this to me.

**Comment by Garima-7** [comment]
Sorry Tried but not able to fix it.

**Comment by ashutoshkrris** [comment]
@Garima-7 No problem
@purohitmansi Would you like to work on it?

**Comment by purohitmansi** [comment]
Yes, I want to give it a try.

**Comment by ashutoshkrris** [comment]
Sure, @purohitmansi 
Assigning it to you.

**Comment by dilroseR** [comment]
i would like to work on this

**Comment by ashutoshkrris** [comment]
@purohitmansi any update?
If you are not able to do it, @dilroseR wants to work on it

**Comment by purohitmansi** [comment]
Go ahead

**Comment by ashutoshkrris** [comment]
@dilroseR do you have prior experience with django?

**Comment by dilroseR** [comment]
> @dilroseR do you have prior experience with django?

yes i've worked with django.. but this issue is fixing the card size,right?

**Comment by ashutoshkrris** [comment]
yeah.
I just wanted to confirm, so that the project setup would be easier for you.

> > @dilroseR do you have prior experience with django?
> 
> yes i've worked with django.. but this issue is fixing the card size,right?

**Comment by ashutoshkrris** [comment]
> yeah.
> I just wanted to confirm, so that the project setup would be easier for you.
> Also, you are a DCP participant, right?
> 
> > > @dilroseR do you have prior experience with django?
> > 
> > 
> > yes i've worked with django.. but this issue is fixing the card size,right?

**Comment by dilroseR** [comment]
> > yeah.
> > I just wanted to confirm, so that the project setup would be easier for you.
> > Also, you are a DCP participant, right?
> > > > @dilroseR do you have prior experience with django?
> > > 
> > > 
> > > yes i've worked with django.. but this issue is fixing the card size,right?

yes a DCP participant

**Comment by ashutoshkrris** [comment]
Cool, I am assigning you the issue

**Comment by dilroseR** [comment]
> Cool, I am assigning you the issue

Sorry, not able to solve the issue.
